From 2e3d2d8896173631a3e21f486292259efc42a2c3 Mon Sep 17 00:00:00 2001
From: Lin Fuzhen <fuzhen.lin@freescale.com>
Date: Tue, 19 Jun 2012 10:37:45 +0800
Subject: [PATCH 1627/2463] ENGR00214081-1 Input: Remove redefined Item in
 Kconfig And Makefile

Remove redefined Item in Kconfig And Makefile for MAX11081

Signed-off-by: Lin Fuzhen <fuzhen.lin@freescale.com>
---
 drivers/input/touchscreen/Kconfig  |   12 ------------
 drivers/input/touchscreen/Makefile |    1 -
 2 files changed, 0 insertions(+), 13 deletions(-)

diff --git a/drivers/input/touchscreen/Kconfig b/drivers/input/touchscreen/Kconfig
index 7942248..7cf975a 100755
--- a/drivers/input/touchscreen/Kconfig
+++ b/drivers/input/touchscreen/Kconfig
@@ -767,16 +767,4 @@ config TOUCHSCREEN_DA9052
         help
           Say y here to support the touchscreen found on
           Dialog Semiconductor DA9052 PMIC
-
-config TOUCHSCREEN_MAX11801
-       tristate "MAX11801 based touchscreens"
-       depends on I2C
-       help
-        Say Y here if you have a MAX11801 based touchscreen
-        controller
-
-        If unsure, say N.
-
-        To compile this driver as a module, choose M here: the
-	  module will be called max11801_ts
 endif
diff --git a/drivers/input/touchscreen/Makefile b/drivers/input/touchscreen/Makefile
index 05155e0..e614512 100755
--- a/drivers/input/touchscreen/Makefile
+++ b/drivers/input/touchscreen/Makefile
@@ -29,7 +29,6 @@ obj-$(CONFIG_TOUCHSCREEN_FUJITSU)	+= fujitsu_ts.o
 obj-$(CONFIG_TOUCHSCREEN_INEXIO)	+= inexio.o
 obj-$(CONFIG_TOUCHSCREEN_INTEL_MID)	+= intel-mid-touch.o
 obj-$(CONFIG_TOUCHSCREEN_LPC32XX)	+= lpc32xx_ts.o
-obj-$(CONFIG_TOUCHSCREEN_MAX11801)	+= max11801_ts.o
 obj-$(CONFIG_TOUCHSCREEN_MC13783)	+= mc13783_ts.o
 obj-$(CONFIG_TOUCHSCREEN_MCS5000)	+= mcs5000_ts.o
 obj-$(CONFIG_TOUCHSCREEN_MIGOR)		+= migor_ts.o
-- 
1.7.7.4

